frontend/src/Components/MainContainer/MainContainer.module.css

```diff
@@ -89,6 +91,43 @@
   margin-top: 23px;
 }
 
+@media screen and (max-width: 1280px) and (max-height: 720px) {
+  .paragraph {
+    font-size: 28px;
+  }
+
+  .title {
+    font-size: 60px;
+  }
+
+  .mainContainer {
+    
+    height: 100vh;
```

Comment:
When we talked about using vh and vw, I meant to put them in the class for the background-image. Perhaps do that and test out, whether it work or not. I will do the same

Reply:
They are specifically for the background image for the respective resolution